tests/MongoDB.Driver.Legacy.Tests/Builders/UpdateBuilderTests.cs

```diff
@@ -838,8 +838,8 @@ public void TestRename()
             Assert.Equal(expected, update.ToJson());
         }
 
-        // tests fail on .NET Core 2.1 with out of order key-value-pairs        
-#if !NETCOREAPP2_1
+        // tests fail on .NET Core 2.1 and higher frameworks with out of order key-value-pairs
+#if NET452 && NETCOREAPP1_1
```

Comment:
It looks like all next frameworks will have behavior which is currently problematic for this test

Reply:
This `#if` can't be right. 

Only one of the two target frameworks can be defined at the same time.

Reply:
agreed. My fault

Reply:
done